### PR TITLE
Re-add old CRD versions to unbreak OLM upgrades

### PR DESCRIPTION
**Pull request by lulf**

### CODE REVIEW

**Comment by k-wall** [comment]
"Read old CRD versions"?  Or "Re-add"?

File: templates/crds/addressplans.crd.yaml [hunk 1]
```diff
@@ -72,13 +78,27 @@ spec:
...
-              type: object
-              description: "Restricts message time-to-live (TTL). Applies to address types queue and topic only."
-              properties:
-                maximum:
-                  description: "Imposes a maximum TTL value (in milliseconds) for messages arriving at the address.  Messages arriving with TTL values greater than this value (or no TTL value) will have their expiration adjusted."
-                  type: integer
-                minimum:
-                  description: "Imposes a minimum TTL value (in milliseconds) for messages arriving at the address.  Messages arriving with TTL values less than this value will have their expiration adjusted."
-                  type: integer
+        displayName:
```
Comment (k-wall) [inline templates/crds/addressplans.crd.yaml:81]: I think the intent was to re-add the old top level fields, did you intent to remove the messageTtl fields?

Comment (lulf) [reply templates/crds/addressplans.crd.yaml:81]: Thanks @k-wall , that was an oversight!

**Comment by lulf** [comment]
@k-wall Yes, I will cherry-pick it there.

**Pull request merged**